packages/roosterjs-editor-plugins/lib/plugins/Picker/PickerPlugin.ts

```diff
@@ -288,7 +289,9 @@ export default class PickerPlugin<T extends PickerDataProvider = PickerDataProvi
 
             return hasMatched;
         });
-        return createRange(startPos, endPos) || this.editor.getDocument().createRange();
+        return startPos && endPos
+            ? createRange(startPos, endPos)
```

Comment:
This changes the logic.

createRange() can still return a valid range if endPos is null. But with this change when endPos is null, it will go to the other way.